### PR TITLE
Update CODEOWNERS

### PR DESCRIPTION
**Pull request by bboykov**

### CODE REVIEW

**Review by mihaylovmihail — APPROVED** [review]

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1,2 +1,5 @@
 # This repo is owned by the universal-publishing team, which is  administered by the FT Content Programme Team.
 * @Financial-Times/universal-publishing
+
+# This repo is supported by:
+* @Financial-Times/content-team @Financial-Times/metadata-team
```
Comment (ivan-p-nikolov) [inline .github/CODEOWNERS:5]: This repo is currently  used only for metadata services

Comment (bboykov) [reply .github/CODEOWNERS:5]: Is this going to be the case in the future? Should I change to metadata only? @ivan-p-nikolov

Comment (ivan-p-nikolov) [reply .github/CODEOWNERS:5]: I would suggest to change it to metadata only. And if content or someone else wants to use it in the future than we can add them. @bboykov

Comment (bboykov) [reply .github/CODEOWNERS]: Done.

**Review by ivan-p-nikolov — APPROVED** [review]

**Pull request merged**